### PR TITLE
added a snapchat phishing page

### PR DESCRIPTION
**Pull request by MagicPotato854**

### CODE REVIEW

File: gofish/snapchat/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Log in | Snapchat</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="icon" href="https://static.snapchat.com/accounts-ui/static/1730142817/images/favicon/favicon.png" type="image/png">
```
Comment (zachflower) [inline gofish/snapchat/index.html:8]: Don't hotlink to external images, download this one and include it in your PR

File: gofish/snapchat/index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+</head>
+<body>
+    <div class="container">
+        <div class="login-box">
+            <img src="https://accounts.snapchat.com/accounts/static/images/ghost/snapchat-app-icon.svg" alt="Snapchat Logo" class="logo">
```
Comment (zachflower) [inline gofish/snapchat/index.html:13]: Same re:hotlinking

File: gofish/snapchat/index.html (hunk 1 above)
Comment (MagicPotato854) [reply gofish/snapchat/index.html]: Alright, I got rid of both Hotlinks

File: gofish/snapchat/index.html (hunk 2 above)
Comment (MagicPotato854) [reply gofish/snapchat/index.html]: Same here, resolved hotlinks.

**Pull request merged**